### ISSUE TITLE
Advanced search should contain the correct taggs

### ISSUE THREAD

**Issue by slickrick1983** [legalthings/feedback#56]
Curretly the search does not consist of the correct taggs.

**Comment by slickrick1983** [comment]
PLease use right taggs
![image](https://cloud.githubusercontent.com/assets/8124767/9273815/7647f70e-4290-11e5-9322-c2382f741ca1.png)

**Comment by Sarfaraaz** [comment]
@slickrick1983 Ik neem aan dat ook de tags moeten worden vertaald. Wat moeten die dan worden?
Helemaal bovenaan hebben we "Include" en "Exclude" wat eigenlijk gewoon Show en Hide kon zijn. Is het goed als ik dat verander naar dat?

**Comment by Scoudem** [comment]
De tags zouden oorspronkelijk gefetch moeten worden van de org (of een andere predefined list). En include en exclude zijn nog onderdeel van een oude stijl, moet nog overgezet worden naar selectize...

**Comment by Sarfaraaz** [comment]
Jij was ermee bezig toch? Dan laat ik dat gedeelte als het toch wordt veranderd.

**Comment by Scoudem** [comment]
Ja, alleen de tags veranderen zou goed genoeg moeten zijn voor nu :)

**Comment by slickrick1983** [comment]
@Scoudem  kan jij deze oppakken en  maken zoals besproken tijdens de laatste sprint meeting

**Comment by Scoudem** [comment]
Yes, deze zal ik zo aanpakken

**Comment by Scoudem** [comment]
@slickrick1983 wat zijn precies de correcte tags?
